src/StudyRegistration/GenericRegistrationRequestForm/handleRegisterFormSubmission.ts

```diff
@@ -0,0 +1,111 @@
+import { doesUserHaveRequestPending } from '../utils';
+import { kayakoConfig, hostname, requestorPath } from '../../localconf';
+import { fetchWithCreds } from '../../actions';
+import { createKayakoTicket } from '../../utils';
+
+const KAYAKO_MAX_SUBJECT_LENGTH = 255;
+const handleRegisterFormSubmission = async (
+  formValues,
+  studyUID,
+  setFormSubmissionButtonDisabled,
+  setFormSubmissionStatus,
+  props,
+  studyRegistrationAuthZ,
+  studyNumber,
+  studyName,
+  setReqAccessRequestPending,
+) => {
+  // first, check if there is already a pending request in requestor
+  try {
+    const userHaveRequestPending = await doesUserHaveRequestPending(studyUID);
+    console.log('userHaveRequestPending', userHaveRequestPending);
+    if (userHaveRequestPending) {
+      // there is already a request for this user on this study, display a message
+      // and disable the button
+      setFormSubmissionButtonDisabled(true);
+      setFormSubmissionStatus({
+        status: 'info',
+        text: `There is already a pending request for this study/user combination,
```

Comment:
I only seem to be able to return this response...

Reply:
oh this is because you already have a request for this resource in the requestor DB

if you want to submit this request again you need to interact with the requestor endpoint to delete your existing request. To do that, you need an access token and your request ID, see https://petstore.swagger.io/?url=https://raw.githubusercontent.com/uc-cdis/requestor/master/docs/openapi.yaml#/Manage/delete_request_request__request_id__delete

Reply:
Thanks, per our discussion I am now able to delete requests